tests/diagnostics-stdout-stderr-test-compiler.toit

```diff
@@ -24,34 +24,28 @@ main args:
     directory.rmdir --recursive tmp-dir
 
 run-test program args --expect-stdout/bool:
-  pipes := pipe.fork
-      true                // use_path
-      pipe.PIPE-INHERITED // stdin
-      pipe.PIPE-CREATED   // stdout
-      pipe.PIPE-CREATED   // stderr
+  process := pipe.fork
+      --use-path
+      --create-stdout
+      --create-stderr
       program
       [program] + args
-  stdout-pipe := pipes[1]
-  stderr-pipe := pipes[2]
-  pid  := pipes[3]
   stdout-bytes := #[]
   task::
-    while chunk := stdout-pipe.read:
-      stdout-bytes += chunk
+    stdout-bytes = process.stdout.in.read-all
```

Comment:
Is this the same behavior? Are we sure we'll have all the output, once `process.wait` returns? Probably.

Reply:
The test passes, and it should intuitively do the same thing.

Reply:
hmm.
The test is now failing on Windows. So maybe it's not the same behavior there...